systemd/CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
-set(CMAKE_INSTALL_SYSTEMD_SERVICEDIR "${CMAKE_INSTALL_LIBEXECDIR}/systemd/system"
```

Comment:
good point. we should not name this variable with `CMAKE`. as it is not defined by CMake, but by ourselves.